CHANGES for release v0.7.1 (#512)
Solo5/solo5: CHANGES.md

## v0.7.1 (2022-03-14)

> * Fix syntax error in stubs ld file (#509)

Reviewer: ```suggestion
* Use `note.not_openbsd` instead of `note.not-openbsd` as section name in `solo5-stubs.ld` to be compatible with GNU ld 2.37 on e.g. Arch Linux, Fedora 35, Alpine 3.15 (@TheLortex, #509)
```

> * Fix `opam-release.sh (#511)

Reviewer: ```suggestion
* Fix `opam-release.sh`: the second variant is `-cross-aarch64` (@dinosaure, #511)
```

> * elftool: generate manifests from stdin or to stdout (#510)

Reviewer: ```suggestion
* elftool: support manifests read from stdin or written to stdout using the `-` character (@ehmry, #510)
```
